ms2query/app_helpers.py

```diff
@@ -475,6 +497,47 @@ def cached_library_matching(documents_query: List[SpectrumDocument],
     return found_matches_s2v
 
 
+def get_nn_predictions(matches: pd.DataFrame,
+                       documents_library: List[SpectrumDocument],
+                       documents_query: List[SpectrumDocument],
+                       cutoff: float = 0.6):
+    """Returns matches with certainty_score column with nn predictions
+
+    Another column is added called probable_match which has a 1 or 0 value
+    corresponding to the prediction being above or below the provided cutoff.
+
+    Args:
+    -------
+    matches:
+        Library matching result of 1 query on library
+    documents_library:
+        Spectra in library
+    documents_query:
+        Spectra in query set. Indices should correspond to indices of matches.
+    cutoff:
+        Value which determines when a match is probable based on prediction
+    """
+    base_name = os.path.split(os.path.dirname(__file__))[0]
+    csv_name = os.path.join(base_name, "model", "model_info.csv")
+    csv_dict = csv2dict(csv_name)
+    max_pmass = float(csv_dict["max_parent_mass"][0])
+    model_file = os.path.join(base_name, "model",
+                              "nn_2000_queries_trimming_simple_10.hdf5")
+    predictions = nn_predict_on_matches(
+        matches, documents_library, documents_query, model_file, max_pmass)
+    matches["certainty_score"] = predictions
+    matches["probable_match"] = [int(pred >= cutoff) for pred in predictions]
```

Comment:
```suggestion
    matches["likely_related"] = [int(pred >= cutoff) for pred in predictions]
```

Reply:
Feel free to pick another name, but I would not display it as ``probable_match`` because "match" will be confused with an actual library match (=same compound).

Reply:
Yes I think that makes a lot of sense :)